src/discord_bot/mod.rs

```diff
@@ -238,32 +254,41 @@ impl EventHandler for Handler {
         _new: Option<Message>,
         event: MessageUpdateEvent,
     ) {
-        if config::get().simple_words_channel != Some(event.channel_id) {
-            return;
+        if config::get().simple_words_channel == Some(event.channel_id) {
+            let Some(content) = event.content else { return; };
+            let Some(author) = event.author else { return; };
+            tokio::runtime::Handle::current().spawn(async move {
+                if let Err(err) = simple_words::on_message(
+                    ctx,
+                    event
+                        .attachments
+                        .as_ref()
+                        .map(|attachments| attachments.is_empty())
+                        == Some(false),
+                    &content,
+                    &author,
+                    event.channel_id,
+                    event.id,
+                )
+                .await
+                {
+                    warn!(
+                        "Error processing message edit from \"{}\" (ID {}): {}",
+                        author.name, author.id, err
+                    );
+                }
+            });
+        } else if GuildStorage::get(match event.guild_id { Some(i) => i, None => return }).await.octal_counter_channel == Some(event.channel_id) {
+            let Some(author) = event.author else { return; };
+            tokio::runtime::Handle::current().spawn(async move {
+                if let Err(err) = event.channel_id.delete_message(&ctx, event.id).await {
```

Comment:
I'm not sure whether we should be deleting incorrect messages from an edit, although I can't really think of any good alternative

Reply:
My reasoning was that any edit of a message will disturb the count, so deletion is probably best.